### PR TITLE
File naming changes

### PR DESCRIPTION
**Pull request by s314cy**
This PR brings a few changes to file naming throughout the project. It includes most of the file renaming made in #244, the goal being to separate the renaming process from the more code-focused changes.

### CODE REVIEW

File: mobile-browser-based-version/src/core/communication/client_builder.ts [hunk 1]
```diff
@@ -1,13 +1,13 @@
...
+import { DecentralizedClient } from './decentralized/decentralized_client'
+import { FederatedClient } from './federated/federated_client'
 import { Platform } from '../../platforms/platform'
+import { Task } from '../task/base/task'
 
 export function getClient (platform: Platform, task: Task, password: string): Client {
   switch (platform) {
     case Platform.decentralized:
-      return new DecentralisedClient(
+      return new DecentralizedClient(
```
Comment (Nacho114) [inline mobile-browser-based-version/src/core/communication/client_builder.ts:10]: we need to decide if to use the British or the US convention ^^

**Review by Nacho114 — APPROVED** [review]
Looks good to me, but there seems to be an issue with the lint-test-build / test-client , can merge once it's fixed.

**Comment by s314cy** [comment]
the CI tests passed, I'm merging :+1:

File: mobile-browser-based-version/src/core/communication/client_builder.ts (hunk 1 above)
Comment (martinjaggi) [reply mobile-browser-based-version/src/core/communication/client_builder.ts:10]: US

**Pull request merged**